app/controllers/admin/books_controller.rb

```diff
@@ -17,7 +17,7 @@ def create
     @book = Book.new(book_params)
     if @book.save
       flash[:success] = t('admin.books.create.flash_success')
-      redirect_to books_path
+      redirect_to admin_books_path
```

Comment:
@paulcsmith. My tests all pass now, but when I investigate behavior in the browser, the behavior is not what I expect. Any advice you have on how to solve this problem would be greatly appreciated. Thanks!